tests/dash/configs/privatelink_config.py

```diff
@@ -137,3 +146,25 @@
         "group_id": ROUTE_GROUP1
     }
 }
+
+METER_POLICY_V4_CONFIG = {
+    f"DASH_METER_POLICY_TABLE:{METER_POLICY_V4}": {
+        "ip_version": "ipv4"
```

Comment:
Hi @mukeshmv , looks like the value should be 1 instead of "ipv4", could you please double check?
https://github.com/sonic-net/sonic-dash-api/blob/master/proto/types.proto#L31

Reply:
Oh I see. I saw "ipv4" being used in ACL tests so assumed that works. Will create PR to fix it. Thanks.